index.js

```diff
@@ -110,6 +110,8 @@ let rules = [
   ["(o|in)k", "onk", 1],
   // frog -> freg
   ["og", "eg", 1],
+  // fire -> fier
+  ["re$", "er", 1]
```

Comment:
this would make 'are' -> 'aer'.

Full list of words affected here https://wordfinder.yourdictionary.com/ending-with/re/